src/launch.md

```diff
@@ -0,0 +1,31 @@
+# Run
+
+Bagua use a built-in module `bagua.distributed.launch` to launch jobs, which has similar usage with Pytorh `torch.distributed.launch`.
```

Comment:
use -> has

module -> tool

has similar usage -> whose usage is similar to

Pytorh -> PyTorch

Reply:
fixed